src/data/media-player.ts

```diff
@@ -194,6 +195,20 @@ export const browseLocalMediaPlayer = (
     media_content_id: mediaContentId,
   });
 
+export const getSignedThumbnailPath = async (
```

Comment:
This method seems very generic. Maybe we can add it as a helper in `data/auth.ts`?

Reply:
Now decided do delete this after lower conversation...